tcpto.c

```diff
@@ -74,6 +74,7 @@ void tcpto_err(ip,flagerr) struct ip_address *ip; int flagerr;
  char *record;
  datetime_sec when;
  datetime_sec lastwhen;
+ datetime_sec firstwhen = 0;
```

Comment:
Why do you move the variable to a greater scope?

Reply:
Just for readability to have all the three variables when, lasthwhen and firstwhen defined near to each other.

Reply:
when you look at the code it is much easier to see that the variable is only used in the narrower scope, and it's closer to the code that actually uses it.